Gopkg.toml

```diff
@@ -23,132 +23,133 @@
+unused-packages = true
 
 [[override]]
-  name = "github.com/coreos/go-systemd"
-  version = "v16"
+name = "github.com/coreos/go-systemd"
+version = "v16"
 
 [[override]]
-  name = "bitbucket.org/ww/goautoneg"
-  # Bitbucket doesn't support git protocol, which causes dep to timeout trying to connect
-  # force to https url instead
-  source = "https://bitbucket.org/ww/goautoneg.git"
+name = "bitbucket.org/ww/goautoneg"
+# Bitbucket doesn't support git protocol, which causes dep to timeout trying to connect
+# force to https url instead
+source = "https://bitbucket.org/ww/goautoneg.git"
 
 [[override]]
-  name = "github.com/hashicorp/serf"
-  revision = "11bb88abf7b17f0b794b51416a9107d781e95f35"
+name = "github.com/hashicorp/serf"
+revision = "11bb88abf7b17f0b794b51416a9107d781e95f35"
 
 [[override]]
-  # target codec version from etcd, since dep doesn't seem to grab it correctly
-  # https://github.com/coreos/etcd/issues/8715#issuecomment-377013707
-  name = "github.com/ugorji/go"
-  version = "v1.1.1"
+# target codec version from etcd, since dep doesn't seem to grab it correctly
+# https://github.com/coreos/etcd/issues/8715#issuecomment-377013707
+name = "github.com/ugorji/go"
+version = "v1.1.1"
 
 [[override]]
-  name = "golang.org/x/sys"
-  revision = "78d5f264b493f125018180c204871ecf58a2dce1"
+name = "golang.org/x/sys"
+revision = "78d5f264b493f125018180c204871ecf58a2dce1"
 
 [[override]]
-  name = "github.com/syndtr/gocapability"
-  revision = "33e07d32887e1e06b7c025f27ce52f62c7990bc0"
+name = "github.com/syndtr/gocapability"
+revision = "33e07d32887e1e06b7c025f27ce52f62c7990bc0"
 
 [[constraint]]
-  name = "github.com/containerd/console"
-  revision = "cb7008ab3d8359b78c5f464cb7cf160107ad5925"
+name = "github.com/containerd/console"
+revision = "cb7008ab3d8359b78c5f464cb7cf160107ad5925"
 
 [[constraint]]
-  name = "github.com/cenkalti/backoff"
-  version = "2.0.0"
+name = "github.com/cenkalti/backoff"
+version = "2.0.0"
 
 [[constraint]]
-  name = "github.com/coreos/etcd"
-  version = "3.3.3"
+name = "github.com/coreos/etcd"
+version = "3.3.3"
 
 [[constraint]]
-  name = "github.com/docker/docker"
-  version = "=1.9.1"
+name = "github.com/docker/docker"
+version = "=1.9.1"
 
 [[constraint]]
-  branch = "master"
-  name = "github.com/gravitational/configure"
+branch = "master"
+name = "github.com/gravitational/configure"
 
 [[constraint]]
-  branch = "master"
-  name = "github.com/gravitational/coordinate"
+branch = "master"
+name = "github.com/gravitational/coordinate"
 
 [[constraint]]
-  branch = "master"
-  name = "github.com/gravitational/go-udev"
+branch = "master"
+name = "github.com/gravitational/go-udev"
 
 [[constraint]]
-  name = "golang.org/x/net"
-  branch = "release-branch.go1.9"
+branch = "release-branch.go1.9"
+name = "golang.org/x/net"
 
 [[constraint]]
-  name = "github.com/gravitational/satellite"
-  version = "=5.5.22"
+name = "github.com/gravitational/satellite"
+#version = "=5.5.22"
+branch = "roman/5.5/dircheck"
```

Comment:
Sorry about all these whitespace changes, my editor decided to auto-format the file. The only real change here is this one to revendor satellite.

Reply:
One could add `?w=1` to URL to skip whitespace only changes. https://github.com/gravitational/planet/pull/727/files?w=1

Reply:
Huh, nice trick!